vm-migration/Cargo.toml

```diff
@@ -13,3 +13,4 @@ serde_json = "1.0.79"
 versionize = "0.1.6"
 versionize_derive = "0.1.4"
 vm-memory = { version = "0.7.0", features = ["backend-mmap", "backend-atomic"] }
+object = "0.28.3"
```

Comment:
I don't really want to add a new dependency if we can help it. There all ELF constants available in linux-loader which is something we already depend upon: https://github.com/rust-vmm/linux-loader/blob/main/src/loader_gen/x86_64/elf.rs

Reply:
The `U64` and `U32` types are for endianess. In Cloud Hypervisor we only support LE and will only support LE so I don't think you should worry about that.

Reply:
Well, actually I met a packed problem when saving elf note header and U32 and U64 solved that. Next I will use elf type in linux-loader to try.

Reply:
Emmm, module `loader_gen` is private, we can't use the elf definition in it.
Any suggestions? @rbradford

Reply:
It's `use pub` exposed as `linux_loader::elf` so e.g. `use linux_loader::elf::Elf64_Phdr;`

Reply:
That works, thx.